performance-tests/bench_2/common/util.h

```diff
@@ -23,7 +24,8 @@ unsigned Bench_Common_Export get_option_argument_uint(int& i, int argc, ACE_TCHA
 std::string Bench_Common_Export join_path(const std::string& arg);
 template <typename... Args>
 std::string join_path(const std::string& arg, Args... args) {
-  return arg + (arg.back() == ACE_DIRECTORY_SEPARATOR_CHAR ? "" : ACE_DIRECTORY_SEPARATOR_STR) + join_path(args...);
+  ACE_TString temp(ACE_TEXT_CHAR_TO_TCHAR(arg.c_str()));
+  return arg + (temp[temp.length() - 1] == ACE_DIRECTORY_SEPARATOR_CHAR ? "" : ACE_TEXT_ALWAYS_CHAR(ACE_DIRECTORY_SEPARATOR_STR)) + join_path(args...);
```

Comment:
So I looked this up and there are `char` versions of these: `ACE_DIRECTORY_SEPARATOR_CHAR_A`` `ACE_DIRECTORY_SEPARATOR_STR_A`.

Reply:
Gotcha.